gql/client.py

```diff
@@ -31,22 +32,33 @@ def __init__(
         assert not (
             type_def and introspection
         ), "Cannot provide introspection type definition at the same time."
```

Comment:
Not related to this PR but I think there is a missing connector `and` on the message.

Should be:
`Cannot provide introspection and type definition at the same time.`

Reply:
Indeed, I made the correction